apps/cron/src/notifications/sendGemsPayoutNotifications.ts

```diff
@@ -0,0 +1,153 @@
+import { log } from '@charmverse/core/log';
+import type { PartnerRewardPayout } from '@charmverse/core/prisma-client';
+import { prisma } from '@charmverse/core/prisma-client';
+import { getCurrentSeasonStart, getCurrentSeasonWeekNumber } from '@packages/dates/utils';
+import { sendNotifications } from '@packages/scoutgame/notifications/sendNotifications';
+import { getClaimablePoints } from '@packages/scoutgame/points/getClaimablePoints';
+import { formatUnits } from 'viem';
+
+const fontFamily = 'Arial, sans-serif';
+const fontColor = '#000';
+
+const partnerRewardsRecord: Record<string, { name: string; partnerLink?: string; icon: string; chain: string }> = {
+  octant_base_contribution: {
+    chain: 'Base',
+    name: 'Octant & Base Contribution',
+    partnerLink: 'https://scoutgame.xyz/info/partner-rewards/octant',
+    icon: 'https://scoutgame.xyz/images/crypto/usdc.png'
+  },
+  optimism_new_scout: {
+    chain: 'Optimism',
+    name: 'Optimism New Scout',
+    partnerLink: 'https://scoutgame.xyz/info/partner-rewards/optimism',
+    icon: 'https://scoutgame.xyz/images/crypto/op.png'
+  },
+  optimism_referral_champion: {
+    chain: 'Optimism',
+    name: 'Optimism Referral Champion',
+    partnerLink: 'https://scoutgame.xyz/info/partner-rewards/optimism',
+    icon: 'https://scoutgame.xyz/images/crypto/op.png'
+  }
+};
+
+function formatPartnerRewardPayout(
+  openingStatement: string,
+  wallets: {
+    partnerRewardPayouts: Pick<PartnerRewardPayout, 'amount'>[] &
+      {
+        payoutContract: {
+          partner: string;
+          tokenDecimals: number;
+        };
+      }[];
+  }[]
+) {
+  if (wallets.length === 0) {
+    return '';
+  }
+
+  let html = `<p style="padding-top: 5px; font-family: ${fontFamily}; color: ${fontColor};">${openingStatement}:</p><ul>`;
```

Comment:
@Devorein going forward, we should still use react for these templates, this is very fragile. u can just render the react on the server-side